docker/data-science/notebooks/ch07/2.semantic-search.ipynb

```diff
@@ -634,58 +603,68 @@
      "output_type": "stream",
      "text": [
       "bbq: {'category': 'Barbeque', 'term_vector': 'bbq^0.9191 ribs^0.6186 pork^0.5991'}\n",
-      "korean bbq: {'category': 'Korean', 'term_vector': 'bbq^0.9052 korean^0.8641 pork^0.6079'}\n",
+      "korean bbq: {'category': 'Korean', 'term_vector': 'korean^0.8095 bbq^0.7241 banchan^0.584'}\n",
       "lasagna: {'category': 'Italian', 'term_vector': 'lasagna^0.9193 alfredo^0.3992 pasta^0.3909'}\n",
       "karaoke: {'category': 'Karaoke', 'term_vector': 'karaoke^0.9193 sing^0.6423 songs^0.5256'}\n",
-      "drive through: {'category': 'Fast Food', 'term_vector': 'through^0.8999 drive^0.8613 thru^0.6118'}\n"
+      "drive through: {'category': 'Fast Food', 'term_vector': 'drive^0.8273 through^0.7593 thru^0.5107'}\n"
```

Comment:
same as last comment. The results order / scoring has changed. I don't think it should have.